### PR TITLE
fix(NavigationBarLogo): alignment and Blau's size

### PR DESCRIPTION
**Pull request by yceballost**
NavigationBarLogo component has two problems fixed:
- Blau logo size was wrong
- Logo wasn't aligned with the center

An extra ball, I removed evenodd from svgs and I exported the logos with the same dimensions as we have in Figma

### CODE REVIEW

**Review by pladaria — APPROVED** [review]
gr8

File: src/navigation-bar.tsx [hunk 1]
```diff
@@ -138,25 +109,40 @@ type NavigationBarLogoProps = {
...
+            break;
         case 'Blau':
-            return <BlauLogo size={size} />;
+            logo = <BlauLogo size={size} />;
+            break;
         default:
-            return null;
+            logo = null;
     }
+    return <div className={classes.logoContainer}>{logo}</div>;
```
Comment (atabel) [inline src/navigation-bar.tsx:145]: hmm... not sure if this is the best place for this div
I'd move it to the header, this component should just return an svg

Comment (yceballost) [reply src/navigation-bar.tsx:145]: How can I resolve this alignment problem?
image

I can't found other alternative 🤔

https://ibit.ly/Qlrn

**Review by atabel — APPROVED** [review]

**Pull request merged**